rest/blip_api_test.go

```diff
@@ -219,7 +219,7 @@ func TestContinuousChangesSubscription(t *testing.T) {
 	subChangesResponse := subChangesRequest.Response()
 	assert.Equals(t, subChangesResponse.SerialNumber(), subChangesRequest.SerialNumber())
 
-	for i := 1; i < 1500; i++ {
+	for i := 1; i < 150000; i++ {
```

Comment:
This change made the test reliably repro the issue, and it still ran in  a few seconds for me locally.